less/_components/classes.less

```diff
@@ -83,28 +83,19 @@ img.desaturate {
   border-color: @brand-warning;
 }
 
-.textfield-underline {
-  border-top:none;
-  border-left:none;
-  border-right:none;
-  border-bottom:1px solid inherit;
-  background-color:transparent;
-  border-radius:0;
-  box-shadow:none;
-  margin:10px auto;
-  color: inherit;
-  // Do not group these rules
-  &:-moz-placeholder { /* Firefox 18- */
-    color: inherit;
+// Example: project page search
+.input-group {
```

Comment:
I'm moving it inside here, since it's a general input class, not just for project discover

Reply:
The problem now is that this will affect all .input-group on the site and we only want to modify the jumbotron...